test/core/container.jl

```diff
@@ -44,21 +42,38 @@ include(dir*"/test/test_utils/AllUtils.jl")
         particles = [Trace(fpc, model, spl, Turing.VarInfo()) for _ in 1:3]
         pc = ParticleContainer(fpc, particles)
 
-        @test getweights(pc) == [1/3, 1/3, 1/3]
-        @test logZ(pc) ≈ log(3)
-        @test pc.logE ≈ log(1)
+        # Initial weights and likelihood.
+        weights = getweights(pc)
+        lz = logZ(pc)
+        @test weights == [1/3, 1/3, 1/3]
+        @test iszero(lz)
 
-        @test consume(pc) == log(1)
+        # Propagate particles.
+        propagate!(pc)
+        @test getweights(pc) == weights
```

Comment:
I'm confused here. Shouldn't new weights be different from old weights, since no resampling is performed in `propagate!`?

Reply:
OK, I forgot that there was no observation in the model. For the purpose of correctly testing `propagate!`, maybe consider adding some likelihood in the test?